cpg_infra/billing_aggregator/aggregate/seqr.py

```diff
@@ -312,18 +312,20 @@ def migrate_entries_from_bq(
             service, sku, usage_start_time, usage_end_time, labels, system_labels,
             location, export_time, cost, currency, currency_conversion_rate, usage,
             credits, invoice, cost_type, adjustment_info
-        FROM `{GCP_BILLING_BQ_TABLE}`
+        FROM @billing_table
```

Comment:
Now that this doesn't have placedholders, you could remove the `f` bit of the `f"string"`

Reply:
Will do.